paddle/fluid/ir_adaptor/translator/op_translator.cc

```diff
@@ -1279,76 +1315,32 @@ struct MulOpTranscriber : public OpTranscriber {
       const std::string& normalized_op_name,
       const OpInputInfoList& input_infos,
       pir::Block* block) override {
-    int x_num_col_dims = paddle::get<int>(op_desc.GetAttr("x_num_col_dims"));
-    int y_num_col_dims = paddle::get<int>(op_desc.GetAttr("y_num_col_dims"));
+    const int x_num_col_dims =
+        PADDLE_GET_CONST(int, op_desc.GetAttr("x_num_col_dims"));
+    const int y_num_col_dims =
+        PADDLE_GET_CONST(int, op_desc.GetAttr("y_num_col_dims"));
+
+    ValueInfo x_info =
+        getMatmulInfo(op_desc, op_desc.Input("X", true), param_map, "X");
+
+    const std::vector<int64_t>& x_shape = std::get<0>(x_info);
+    const dialect::DenseTensorType& x_tensor_type = std::get<1>(x_info);
+    pir::OpResult& x_value = std::get<2>(x_info);
```

Comment:
```suggestion
    cosnt auto& [x_shape, x_tensor_type, x_value] = x_info;
```
Try structured binging  to simplify this.